src/web/views.py

```diff
@@ -1,21 +1,46 @@
+from typing import Any
 from django.http import HttpRequest, HttpResponse
 from django.shortcuts import render
 from django.utils import timezone
 
+from django.views.generic import DetailView
+from handyhelpers.views.gui import HandyHelperListView, HandyHelperIndexView
 from handyhelpers.views.htmx import GenericHtmxView
 
-# import models
 from web.models import Event, TechGroup
 
 
-# Create your views here.
-def index(request: HttpRequest) -> HttpResponse:
-    return render(request, "web/index.html")
+class Index(HandyHelperIndexView):
+    title = "Spokane Tech"
+    subtitle = "Index of Spokane's Tech User Groups"
+    base_template = "spokanetech/base.html"
 
+    def __init__(self, **kwargs: Any) -> None:
```

Comment:
An init and super shouldn't be required here. Did this not work with just the item_list? I've used this a ton (though never with django v5).

Reply:
I didn't try this without `__init__`. I figured executing db calls in a class definition was a bad idea.